nmostesting/suites/IS1102Test.py

```diff
@@ -13,16 +13,1402 @@
+                if "height" in flow_components[i]:
+                    heights.append(
+                        {
+                            "height" + str(i): flow_components[i]["height"],
+                            "err" + str(i): None,
+                        }
+                    )
+                else:
+                    heights.append(
+                        {"height" + str(i): None, "err" + str(i): "not defined"}
+                    )
+
+            if (
+                heights[0]["err0"] is not None
+                or heights[1]["err1"] is not None
+                or heights[2]["err2"] is not None
+            ):
+                return "invalid array of video components"
+
+            if (
+                widths[0]["width0"] == widths[1]["width1"]
+                and widths[0]["width0"] == widths[2]["width2"]
+                and heights[0]["height0"] == heights[1]["height1"]
+                and heights[0]["height0"] == heights[2]["height2"]
+            ):
+                return "YCbCr-4:4:4"
+
+            if (
+                widths[0]["width0"] == 2 * widths[1]["width1"]
+                and widths[0]["width0"] == 2 * widths[2]["width2"]
+                and heights[0]["height0"] == heights[1]["height1"]
+                and heights[0]["height0"] == heights[2]["height2"]
+            ):
+                return "YCbCr-4:2:2"
+
+            if (
+                widths[0]["width0"] == 2 * widths[1]["width1"]
+                and widths[0]["width0"] == 2 * widths[2]["width2"]
+                and heights[0]["height0"] == 2 * heights[1]["height1"]
+                and heights[0]["height0"] == 2 * heights[2]["height2"]
+            ):
+                return "YCbCr-4:2:0"
+
+        return "invalid array of video components"
+
+    def get_another_grain_rate(self, grain_rate):
+        numerator = grain_rate["numerator"]
+        denominator = grain_rate["denominator"]
+        if (numerator == 30 or numerator == 25) and denominator == 1:
+            return {"numerator": numerator * 2, "denominator": 1}
+        if (numerator == 60 or numerator == 50) and denominator == 1:
+            return {"numerator": numerator / 2, "denominator": 1}
+        if numerator == 24 and denominator == 1:
+            return {"numerator": 30, "denominator": 1}
+        if (numerator == 30000 or numerator == 25000) and denominator == 1001:
+            return {"numerator": numerator * 2, "denominator": 1001}
+        if (numerator == 60000 or numerator == 50000) and denominator == 1001:
+            return {"numerator": numerator / 2, "denominator": 1001}
+        return "grain_rate not valid"
+
+    def get_another_sample_rate(self, sample_rate):
+        numerator = sample_rate["numerator"]
+        if numerator == 48000:
+            return {"numerator": 44100}
+        if numerator == 44100:
+            return {"numerator": 48000}
+        if numerator == 96000:
+            return {"numerator": 4800}
```

Comment:
missing a 0 => 48000

Reply:
Why is this marked resolved, @gwgeorgea ?

Reply:
Yes, there was a misunderstanding on Alain's comment. It is now fix in PR-783